omp/matrix/dense_kernels.cpp

```diff
@@ -244,7 +245,69 @@ GKO_INSTANTIATE_FOR_EACH_VALUE_AND_INDEX_TYPE(
 template <typename ValueType, typename IndexType>
 void convert_to_sellp(std::shared_ptr<const OmpExecutor> exec,
                       matrix::Sellp<ValueType, IndexType> *result,
-                      const matrix::Dense<ValueType> *source) NOT_IMPLEMENTED;
+                      const matrix::Dense<ValueType> *source)
+{
+    auto num_rows = result->get_size().num_rows;
+    auto num_cols = result->get_size().num_cols;
+    auto vals = result->get_values();
+    auto col_idxs = result->get_col_idxs();
+    auto slice_lengths = result->get_slice_lengths();
+    auto slice_sets = result->get_slice_sets();
+    auto slice_size = (result->get_slice_size() == 0)
+                          ? matrix::default_slice_size
+                          : result->get_slice_size();
+    auto stride_factor = (result->get_stride_factor() == 0)
+                             ? matrix::default_stride_factor
+                             : result->get_stride_factor();
+    int slice_num = ceildiv(num_rows, slice_size);
+    slice_sets[0] = 0;
+
+#pragma omp parallel for
```

Comment:
This is fine for now, but you may really lack parallelism depending on how many slices you typically have. For small cases this may be rather important. Maybe parallelizing the inner loops, if possible, would yield better results.